Example/Example.xcodeproj/project.pbxproj

```diff
@@ -179,7 +179,7 @@
 				TargetAttributes = {
 					391BB4731FBD944500D798E4 = {
```

Comment:
Could you briefly describe in the commit message, the reason behind this change even if it looks obvious to you